blog-cse/2023-02-22-application-update.md

```diff
@@ -0,0 +1,65 @@
+
+CSE will include a set of more than twenty First Seen Rules out of the box. These rules can be tuned and customized like any other rule type, and custom First Seen Rules can also be created.
+
+For more information about how to use First Seen Rules, see the online [documentation](https://help.sumologic.com/docs/cse/rules/write-first-seen-rule/). You can also see an introduction to the feature by navigating to a new First Seen Rule in the CSE UI.
+
+### Entity Timeline
+
+Another new feature that will help analysts investigate unusual activity with user accounts is the Entity Timeline:
+
+<img src={useBaseUrl('img/release-notes/cse/entity-timeline-rn.png')} alt="The Entity Timeline"/>
+
+This feature visualizes all activity for a user – including all normalized records – in an easy-to-read timeline, eliminating the need to perform manual record searches.
+
+Related actions are grouped together and Signals and Insights generated on that user are also displayed in the timeline with the relevant record(s). Actions can be clicked on to see a more detailed set of information, and full details can be easily opened in a new tab. 
+
+The feature can be found on the new **Timeline** tab on each Username Entity’s Detail page with quick links from Signal and Insight detail pages (located with the Entity summaries). It is only available for the **Username** Entity type at this time.
+
+For more information about how to use the Entity Timeline, see the online [documentation](https://help.sumologic.com/docs/cse/records-signals-entities-insights/view-manage-entities/#about-the-entity-timeline-tab).
+
+### Minor Changes and Enhancements ###
+
+* [Updated] Entities listed in the Signals index (`sec_signal`) now include `criticality` and `suppressed` attributes (which reflect the state of those Entities when the Signal was generated).
+* [New] The CSE API now supports searching the Threat Intelligence data by `sourceName`. 
+* [Updated] The Threat Intelligence API `GetThreatIntelIndicators` endpoint now supports data sets of more than 10,000 indicators.
+* [Updated] The Insights API now supports searching (filtering) by confidence score.
+* [Updated] CSE now supports up to 1000 inventory-based Entity Groups (the previous limit was 50).
+* [Updated] When viewing an Insight, a label is displayed that indicates the source. When an Insight is generated by a Custom Insight, it will now say **Custom Insight (Rule)** instead of **Rule**) and **Custom Insight (Signal)** (instead of **Signal**) to reduce confusion with Insights generated by the Insight Algorithm through standard Rules and Signals. 
```

Comment:
```suggestion
* [Updated] When viewing an Insight, a label is displayed that indicates the source. When an Insight is generated by a Custom Insight, it will now say **Custom Insight (Rule)** (instead of **Rule**) and **Custom Insight (Signal)** (instead of **Signal**) to reduce confusion with Insights generated by the Insight Algorithm through standard Rules and Signals. 
```